bluesky-social/social-app
# [Video] Error banner improvements

- Stop showing errors when cancelling upload (ignore AbortError)
- Extract ErrorBanner to it's own component
- Show upload `jobId` if appropriate
- Smoothly animate banner appearance

Transition

https://github.com/user-attachments/assets/166497ad-bf10-4711-842d-e08e07e7f9aa

# Test plan

1. Make sure cancelling the upload no longer makes an error appear
2. Check LayoutTransition doesn't break anything - had to disable on web, make sure to check Android
